src/findUnused.test.js

```diff
@@ -20,7 +22,7 @@ describe('findUnused', () => {
     expect(await findUnused(secrets)).toEqual([])
   })
   it('Returns empty array if some of the secrets are unused', async () => {
```

Comment:
The description of the test suggests that the `findUnused` returns an empty array if some of the secrets is unused, but what I see is that it's not the case. Maybe the test description should be updated?